libraries/AP_RangeFinder/AP_RangeFinder_NRA24_CAN.cpp

```diff
@@ -1,6 +1,6 @@
 #include "AP_RangeFinder_config.h"
 
-#if AP_RANGEFINDER_NRA24_CAN_ENABLED
+#if AP_RANGEFINDER_NRA24_CAN_ENABLED || AP_RANGEFINDER_HEXSOONRADAR_ENABLED
```

Comment:
This is really, really ugly.

```suggestion
#if AP_RANGEFINDER_NRA24_CAN_DRIVER_ENABLED
```

... and set that if either of AP_RANGEFINDER_NRA24_CAN_ENABLED or AP_RANGEFINDER_HEXSOONRADAR_ENABLED is set.

I think that should simplify AP_RangeFinder.cpp a lot

Reply:
... would be nice to find something generic to take the palce of `AP_RANGEFINDER_NRA24_CAN_DRIVER_ENABLED` here, but this is better than starting to || in ever damned alias for the same device.

Reply:
ok, thanks for the patch.  TBH, I'm not sure I agree it's any better but the discussion at least led me to go back and review again and I had missed another place where the NRA_CAN enum was being used (now fixed).